connect/src/main/java/com/stripe/android/connect/webview/serialization/IntAsRgbHexString.kt

```diff
@@ -0,0 +1,37 @@
+package com.stripe.android.connect.webview.serialization
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+/**
+ * Indicates that the integer should be serialized as a hex color string, e.g. "#FF0000".
+ */
+typealias IntAsRgbHexString = @Serializable(IntAsRgbHexStringSerializer::class) Int
```

Comment:
\nit I think it'd be nice to add a note for posterity about the serialization issue we found, maybe:
```suggestion
/**
 * Indicates that the integer should be serialized as a hex color string, e.g. "#FF0000".
 * Note that typealias's such as this are only serializable when in a class, and cannot be
 * directly serialized on their own. See https://youtrack.jetbrains.com/issue/KT-64920/Json.encodeToString-yields-different-results-depending-on-whether-typealias-is-used#focus=Comments-27-9092520.0-0.
 */
typealias IntAsRgbHexString = @Serializable(IntAsRgbHexStringSerializer::class) Int
```